x509/golang.go

```diff
@@ -1,4 +1,6 @@
-//go:build windows
+//go:build go1.15 && !bashpki
```

Comment:
so `go1.15` means greater than or equal to go1.15?

Reply:
i.e. our go.mod is pinned on go1.19 already

Reply:
Yes and Yes

Maybe this is redundant, and we only need the `bashpki` flag.
In the initial version I thought to have no `bashpki`, and needed some flag to disable the bash implementation.